openmrs-19/src/main/java/org/motechproject/openmrs19/service/impl/OpenMRSPersonServiceImpl.java

```diff
@@ -93,7 +93,20 @@ public Person createPerson(String configName, String givenName, String familyNam
     public Person updatePerson(String configName, Person person) {
         try {
             Config config = configService.getConfigByName(configName);

```

Comment:
we should add validation for person. I suppose that only uuid is required here, am I right ?